.codecov.yml

```diff
@@ -19,5 +19,5 @@ comment:
   require_head: yes
 
 ignore:
-  - "gen-snapshot-tests.js"
+  - "utils/*.js"
```

Comment:
I think this is the right thing to do, unless we're hoping to cover the tools too.